features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/customreaction/picker/EmojiPicker.kt

```diff
@@ -53,105 +55,83 @@ fun EmojiPicker(
     modifier: Modifier = Modifier,
 ) {
     val coroutineScope = rememberCoroutineScope()
-    val categories = state.categories
-    val pagerState = rememberPagerState(pageCount = { EmojibaseCategory.entries.size })
-
+    val pagerState = rememberPagerState(pageCount = { state.categories.size })
     Column(modifier) {
-        SearchBar(
```

Comment:
Why are you removing the search bar? 😞

Reply:
Oh, it must be a rebase issue, thanks for spotting it!

Reply:
This should be fixed now. Sorry for the misunderstanding!